tools/releaser/bin/releaser.dart

```diff
@@ -105,24 +101,25 @@ void main(List<String> arguments) async {
   }
 
   final commands = <Command>[
-    ValidateReleaseCommand(),
-    CreateBranchCommand(choreBranch),
-    UpdateVersionsCommand(),
-    CommitChangesCommand(
-        'chore: Preparing for release of ${commandArgs.packageName} ${commandArgs.version}.'),
-    CreateReleaseBranchCommand(),
-    RemoveDependencyOverridesCommand(),
-    RemovePodOverridesCommand(),
-    UpdateGradleFilesCommand(),
-    CommitChangesCommand(
-      'chore: Remove dependency overrides for release of ${commandArgs.packageName} ${commandArgs.version}.',
-      noChangesOkay: true,
-    ),
-    ValidatePublishDryRun(),
-    SwitchBranchCommand(choreBranch),
-    BumpVersionCommand(versionBumpType),
-    CommitChangesCommand(
-        'chore: Bump version of ${commandArgs.packageName} to next potential release.'),
+    // ValidateReleaseCommand(),
```

Comment:
these are commented instead of being deleted just for the opportunity to bring them back in the future?

Reply:
Nope that's a mistake....